pkg/vdr/key/creator.go

```diff
@@ -88,10 +97,82 @@ func (v *VDR) Create(keyManager kms.KeyManager, didDoc *did.Doc,
 	return &did.DocResolution{DIDDocument: createDoc(publicKey, keyAgr, didKey)}, nil
 }
 
+func getKeyTypeAndVerMethodType(kt kms.KeyType) (kms.KeyType, string) {
+	var (
+		keyType kms.KeyType
+		vmType  string
+	)
+
+	switch kt {
```

Comment:
Probably better to have a map.

Reply:
a new function called `getVerMethodType` is added and has a map for verification method type